support/hololens/ServoApp/ServoControl/Servo.cpp

```diff
@@ -87,12 +87,16 @@ const char *prompt_input(const char *message, const char *default,
   }
 }
 
+void on_devtools_started(const unsigned int port) {
+  return sServo->Delegate().OnDevtoolsStarted(port);
+}
+
 Servo::Servo(hstring url, hstring args, GLsizei width, GLsizei height,
              float dpi, ServoDelegate &aDelegate)
     : mWindowHeight(height), mWindowWidth(width), mDelegate(aDelegate) {
 
   capi::CInitOptions o;
-  hstring defaultPrefs = L" --pref dom.webxr.enabled";
+  hstring defaultPrefs = L" --pref dom.webxr.enabled  --devtools=0";
```

Comment:
For now, remove this change. I will add support for Windows/HoloLens in another PR.